src/main.ts

```diff
@@ -16,7 +16,7 @@ function getQIIME2Channel(buildTarget: string) {
 
     case 'release':
     default:
-      return 'qiime2/label/r2020.11'
+      return 'qiime2/label/r2021.2'
```

Comment:
The "staging" case just above this needs to be updated, too (to 2021.4) - thanks!

Reply:
Great, thanks! Unfortunately you'll have to rerender the output, using `npm run all`.